account_controllers/controllers.js

```diff
@@ -176,20 +223,111 @@ module.exports = {
 
     editOperationHours: async (req, res) => {
       try {
-        const { id } = req.params;
         const { operation_hours } = req.body;
-        const { error } = await supabase
+        const { id } = req.params;
+        const [startHour, endHour] = operation_hours.split('-').map(Number);
+
+        if (
+          startHour < 0 ||
+          startHour > 24 ||
+          endHour < 0 ||
+          endHour > 24 ||
+          startHour >= endHour
```

Comment:
Comment: this condition does not necessarily work if a garage is open from 8pm to 2am?

Reply:
i think the intention and we'll have a check on the front end for the valet admin is that lower hour as first element, then the higher hour. The backend will also do a check for this to ensure startHour >= endHour as well and make sure the numbers are not out of  bounds. It'll be on a 24 hour time scale and its positioned as '5-17' for instance